dex/fuzz/fuzz_targets/multiple_orders.rs

```diff
@@ -392,6 +400,9 @@ fn run_action<'bump>(
                     if instruction.self_trade_behavior == SelfTradeBehavior::AbortTransaction => {}
                 DexError::ErrorCode(DexErrorCode::WrongOrdersAccount)
                     if owner.closed_open_orders => {}
+                // Open orders account hasn't been initialized.
+                DexError::ErrorCode(DexErrorCode::InvalidOpenOrdersAuthority)
+                    if market_accounts.open_orders_authority.is_some() => {}
```

Comment:
Given the current fuzz tests, this should never happen unless there's a bug, right? So why swallow the error instead of panicking?

Reply:
Permissioned markets currently don't allow implicit open orders initialization via `NewOrderV3`, in which case this will be returned.

The execution is

* https://github.com/project-serum/serum-dex/blob/master/dex/src/state.rs#L1815-L1816
* https://github.com/project-serum/serum-dex/blob/master/dex/src/state.rs#L139-L141